eng/packages/TestOnly.props

```diff
@@ -22,7 +22,7 @@
     <PackageVersion Include="Polly.Testing" Version="8.4.2" />
     <PackageVersion Include="StrongNamer" Version="0.2.5" />
     <PackageVersion Include="System.Configuration.ConfigurationManager" Version="$(SystemConfigurationConfigurationManagerVersion)" />
-    <PackageVersion Include="System.Numerics.Tensors" Version="$(SystemNumericsTensorsVersion)" />
+    <PackageVersion Include="System.Linq.Async" Version="6.0.1" />
```

Comment:
Can we use System.Linq.AsyncEnumerable instead? System.Linq.Async is now legacy.